### ISSUE TITLE
Publish the package to the pypi

### ISSUE THREAD

**Issue by mathbunnyru** [qchateau/conan-center-bot#11]
It will make it much easier to install and use the package

**Comment by qchateau** [comment]
I think it's done ?

https://pypi.org/project/conan-center-bot/

Let me know if it does not work though

**Comment by mathbunnyru** [comment]
That's great to hear!

```
~ > pip3 install conan-center-bot --no-cache-dir
Collecting conan-center-bot
  Downloading conan-center-bot-0.0.3.tar.gz (12 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/local/opt/python@3.9/bin/python3.9 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/y5/g24cdzkd61zfnvjslftfwb000000gn/T/pip-install-wxeyyve2/conan-center-bot/setup.py'"'"'; __file__='"'"'/private/var/folders/y5/g24cdzkd61zfnvjslftfwb000000gn/T/pip-install-wxeyyve2/conan-center-bot/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/y5/g24cdzkd61zfnvjslftfwb000000gn/T/pip-pip-egg-info-ie2wmg53
         cwd: /private/var/folders/y5/g24cdzkd61zfnvjslftfwb000000gn/T/pip-install-wxeyyve2/conan-center-bot/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/y5/g24cdzkd61zfnvjslftfwb000000gn/T/pip-install-wxeyyve2/conan-center-bot/setup.py", line 25, in <module>
        install_requires=get_requires("requirements.txt"),
      File "/private/var/folders/y5/g24cdzkd61zfnvjslftfwb000000gn/T/pip-install-wxeyyve2/conan-center-bot/setup.py", line 7, in get_requires
        with open(filename, "rt") as req_file:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

Doesn't seem to work now properly right now because of requirements.txt missing file

**Comment by mathbunnyru** [comment]
And I think we should add to the readme how to install the bot and use it without `python3 -m ccb` or `python3 -m main.py`.

**Comment by qchateau** [comment]
Should be fixed now

**Comment by mathbunnyru** [comment]
Something is wrong with versioning.

When I do `pip3 install conan-center-bot --upgrade` multiple times, it always reinstalls the package (but it shouldn't).

```
Installing collected packages: conan-center-bot
  Attempting uninstall: conan-center-bot
    Found existing installation: conan-center-bot 0.0.0
    Uninstalling conan-center-bot-0.0.0:
      Successfully uninstalled conan-center-bot-0.0.0
Successfully installed conan-center-bot-0.0.0
```

**Comment by mathbunnyru** [comment]
I think something wrong is going on here:
https://github.com/qchateau/conan-center-bot/blob/master/setup.py#L16

**Comment by mathbunnyru** [comment]
When setup.py runs locally there's no CCB_VERSION env variable, that's why it would always be 0.0.0.

**Comment by mathbunnyru** [comment]
I did `export CCB_VERSION=0.0.4` to check it and after the first reinstallation works well (without reinstalling).

**Comment by qchateau** [comment]
In the end do you think something is wrong ?
When I `pip install conan-center-bot` it seem to install 0.0.4 properly from pypi

**Comment by mathbunnyru** [comment]
Try to install it once again with `--upgrade` flag.

If everything is right, it will say something like `Requirement already up-to-date`.
If not, it will install your package once again, the same version. For me this is the case.

**Comment by qchateau** [comment]
Ah yes, you're right, the package is registered as 0.0.4 but installs as 0.0.0 which confuses pip.

It's fixed with 0.0.5, though I had to flush my pip cache

**Comment by mathbunnyru** [comment]
Confirmed. Works as expected now.
Thanks.
